clinical/report-en.tex

```diff
@@ -0,0 +1,211 @@
+test result, you should contact a doctor or healthcare provider.}
+
+\begin{itemize}
+
+
+\item
+
+  \textbf{If your test says ``Not Detected (Negative)''}, it means our lab did not find the
+  virus that causes COVID-19 in your sample.
+
+  \begin{itemize}
+    \item
+      If you have \textbf{symptoms or have been exposed} to a known positive, \textbf{STAY HOME}.
+    \item
+      Review Public Health – Seattle \& King County’s latest guidance (link below) on how long you need to stay home in quarantine.
+    \item
+      You may still have COVID-19 even with a negative test. The test cannot detect the virus if it is too soon after exposure.
+      Wait 5-7 days after date exposed to get tested unless you develop symptoms.
+    \item
+      If you have no symptoms and no known exposure, you likely do not have COVID-19.
+    \item
+      Continue to practice mask wearing, social distancing, and monitor for symptoms.
+  \end{itemize}
+
+\item
+
+  \textbf{If your test says ``Detected (Positive)''}, it means our
+  lab found the virus that causes COVID-19 in your sample.
+
+  \begin{itemize}
+    \item \textbf{STAY HOME} for 10 days starting from the date symptoms first appeared \textbf{AND} 24 hours with no fever \textbf{AND} until symptoms resolve. Tell anyone you’ve been physically close with to get tested.
+    \item If no symptoms, then stay home 10 days from date of test. You can still spread COVID-19 even if you don’t have symptoms.
+    \item You do NOT need to avoid the people in your house who have already been diagnosed with COVID-19.
+    \item You do NOT need to test again after completing the 10-day isolation period.
+  \end{itemize}
+
+  Your result will be shared with the Washington State and local health
+  departments. You do \textbf{not} need to report these results yourself. The
+  health department may call you to ask questions about your illness and people
+  you may have been around while you were sick.
+
+\item
+
+  \textbf{If your test says ``Inconclusive''}, it means our lab is not certain
+  of your test result. No test is perfect, and an inconclusive result happens
+  when a small amount of SARS-CoV-2 is found in your sample, but not enough for
+  us to say that you have COVID-19.
+
+  Current public health guidelines suggest that you get an additional test, but
+  you should treat this result as a positive and stay home, away from others,
+  even if you don’t have symptoms of COVID-19.
+
+\item
+
+  \textbf{If your test says ``Unable to be tested''}, it means there was either
+  not enough material on your sample for us to test, or there was an error when
+  we ran the test. Since we are not able to test your sample, we could not find
+  whether or not you have the virus that causes COVID-19.
+
+\end{itemize}
+
+\bigskip
+
+\textbf{Visit the King County Public Health website to learn more about how and when to isolate and quarantine:\\\\}
+\includegraphics[height=0.5in,trim=0 0 0 0]{king_county_covid19_qr}
```

Comment:
I don't know if this matters, but when the QR code was bigger my phone recognized it from my screen with the PDF being viewed at 100%. With the smaller QR code I have to make the PDF 125% big for my phone to recognize the code.
If we do end up needing this bigger and need to cut room, removing the itemization of the "About this test" at the very top (the itemized list with one item) might free up some space.

Reply:
Let's err on the safe side and bump it back up to .75 inches. The QR code isnt much good if its not scannable. 
This makes it overflow to 2 pages again. Removing the itemization almost but didn't quite remove a line. Instead, I've introduced the latex package wrapfigure and used it to wrap the text around that qr code. Included in latest push.

Reply:
Oh cool! Would you please put an example of the latest into the Trello card?

Reply:
Sure thing, I've uploaded a new copy there. Here's the command I'm using to generate them:

`./devel/docker-run fill-template --template clinical/report-en.tex --params clinical/example-params.csv --filter 'status_code not in ["not-received", "pending"]' --output "{qrcode}".pdf`

Reply:
That looks so good! The layout really looks good on this template.
My phone scanned the QR code perfectly. Thanks for making that change.